proposal template update (#49)
near/devrel: .github/ISSUE_TEMPLATE/proposal-template.md

---

> # Proposal for: [insert what you are proposing should be worked on in 10 words or fewer]

Reviewer: ```suggestion
# Proposal for: [less than 10 words describing the work to be done]
```

## What is this? 
- 

## Who needs to be involved?
-

> ## How will we know this is done? (Definition of Done)

Reviewer: ```suggestion
## Tasks to complete
[ ] - 
```

What do you think of changing this to "Tasks to complete"?  I like having a list of checkboxes breaking down the steps and indicating when the work is done.
